modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -4478,6 +4478,7 @@ else if (one.required)
         op.requiredParams = requiredParams;
         op.optionalParams = optionalParams;
         op.externalDocs = operation.getExternalDocs();
+        op.hasExternalDocs = op.externalDocs != null;
```

Comment:
You're making changes to `DefaultCodegen` and `CodegenOperation` so you'll also likely need a review from the core team as well (even though this is minor)

Reply:
We could maybe initiate the `hasExternalDocs ` attribute at other places. Thus yes the Core Team is welcome :)
@wing328
@jimschubert
@cbornet 
@jmini
@etherealjoy 
@spacether